op-batcher/batcher/service.go

```diff
@@ -198,7 +198,7 @@ func (bs *BatcherService) initChannelConfig(cfg *CLIConfig) error {
 	}
 
 	switch cfg.DataAvailabilityType {
-	case flags.BlobsType:
+	case flags.BlobsType, flags.AutoType:
```

Comment:
If calldata is more cost-effective, and the batcher has been submitting calldata type data, if the batcher is suddenly restarted, will it revert to submitting blob? I believe that when the batcher starts, it should choose between blob and calldata based on the actual situation, rather than defaulting to blob.